lib/routes/account.js

```diff
@@ -485,15 +487,20 @@ module.exports = function (
           // not performing a sign-in confirmation.
           var shouldSendNewDeviceLoginEmail = config.newLoginNotificationEnabled && requestHelper.wantsKeys(request) && !doSigninConfirmation
           if (shouldSendNewDeviceLoginEmail) {
-            // The response doesn't have to wait for this,
-            // so we don't return the promise.
-            mailer.sendNewDeviceLoginNotification(
-              emailRecord.email,
-              userAgent.call({
-                acceptLanguage: request.app.acceptLanguage,
-                timestamp: Date.now()
-              }, request.headers['user-agent'])
-            )
+            return getGeoData(ip)
```

Comment:
It is safe to remove the `return` here since we don't want to wait for email to be sent before proceeding.